src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java

```diff
@@ -1426,6 +1426,32 @@ public void testFailedToResumeTransaction() throws Exception {
             }
         }
 
+        /**
```

Comment:
can we test all boundary cases here eg. < 1 and > 1 with precision 38, > 38, etc

Reply:
also, test cstmt as in other tests

Reply:
cstmt introduces rounding to 4 decimal places. For cases with maximum precision, I'm sticking with pstmt, for the cases with less than maximum, I've switched to cstmt.

Test cases include: max precision between 0 and 1, max with number greater than 1, less than max precision for a number between 0 and 1, less than max for a number greater than 1.